example/windows/win32_window.cc

```diff
@@ -114,6 +116,9 @@ Win32Window::MessageHandler(HWND hwnd, UINT const message, WPARAM const wparam,
         SetFocus(child_content_);
       }
       return 0;
+    case WM_FONTCHANGE:
+      controller_->OnFontChange();
```

Comment:
I don't know enough about Windows to know if this is a reasonable suggestion or not, but I'm curious if we could forward `WM_FONTCHANGE` to `child_content_` and avoid the need for the special-purpose APIs at the embedding layer. Then we could just handle `WM_FONTCHANGE` in the embedding layer (in fact, you already do, where I commented wondering whether that code was doing anything).

@clarkezone Is that doable? A good/bad idea?

Reply:
Seems reasonable.  I assume WM_FONTCHANGE isn’t automatically propogated to child windows hence the approach here, but manually posting it would avoid the need for the API as you say.

Reply:
Is there a standard way to forwarding windows message? It does not seems like windows provide an api to reroute message to child process. If we want to use sendMessage to forward the message, the embedding will have to start a message loop as well

Reply:
> reroute message to child process

It's not a child process, it's just a different HWND in the same process.

> If we want to use sendMessage to forward the message, the embedding will have to start a message loop as well

Why would that be the case? The child window already receives messages (e.g., mouse events), so clearly the existing message loop is capable of sending messages to that HWND.

Also, the docs for SendMessage say: "If the specified window was created by the calling thread, the window procedure is called immediately as a subroutine." I would expect this is the same thread.

Reply:
i made it too complicated, fixed